modules/transactions.js

```diff
@@ -102,6 +102,8 @@ __private.list = function(filter, cb) {
 	const params = {};
 	const where = [];
 	const allowedFieldsMap = {
+		senderIdOrRecipientId:
+			't_senderId" IN (${senderId:csv}) OR t_recipientId" IN (${recipientId:csv})',
```

Comment:
It seems to be missing double quote in the beginning of `t_senderId` and `t_recipientId`